source/common/http/http1/codec_impl.cc

```diff
@@ -285,8 +282,9 @@ void StreamEncoderImpl::encodeTrailersBase(const HeaderMap& trailers) {
       return HeaderMap::Iterate::Continue;
     });
 
-    flushOutput();
```

Comment:
So we had this here since https://github.com/envoyproxy/envoy/pull/8667/files and at the time it was needed to flush before doing the add (see comments in that PR).  Can you confirm that is no longer necessary?
If it's not necessary I think you can remove the flushOutput after the add as it'll be flushed immediately on exiting the scope, no?

Reply:
get it.